deepsource/client.go

```diff
@@ -30,19 +37,30 @@ func (c Client) GQL() *graphql.Client {
 
 // Returns the jWT which is required for authentication and thus, interacting with the APIs
 func (c Client) GetToken() string {
-	return config.Cfg.Token
+	return c.token
 }
 
 // Returns a new GQLClient
-func New() (*Client, error) {
-	// TODO: Figure out better way to resolve host here
-	apiClientURL := fmt.Sprintf("https://api.%s/graphql/", config.Cfg.Host)
+func New(cp ClientProperties) (*Client, error) {
```

Comment:
Call it Opts since that is a standard convention with a lot of repos.

Reply:
Named it `ClientOpts`. Gives more context.